example/src/index.js

```diff
@@ -1,11 +1,49 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import App from './components/app';
+import './reset.css'
+import './index.css'
 
-const MOUNT_NODE = document.getElementById('app');
+import React from 'react'
+import ReactDOM from 'react-dom'
+import { HashRouter as Router, Switch, Route } from 'react-router-dom'
+import Home from './Home'
+import VerticalBar from './charts/VerticalBar'
+import HorizontalBar from './charts/HorizontalBar'
+import StackedBar from './charts/StackedBar'
+import GroupedBar from './charts/GroupedBar'
+import Line from './charts/Line'
+import MultiAxisLine from './charts/MultiAxisLine'
+import Scatter from './charts/Scatter'
+import Doughnut from './charts/Doughnut'
+import Pie from './charts/Pie'
+import Polar from './charts/Polar'
+import Radar from './charts/Radar'
+import Dynamic from './charts/Dynamic'
+import MultiType from './charts/MultiType'
+import Crazy from './charts/Crazy'
+import ClickEvents from './charts/ClickEvents'
 
-const render = () => {
-    ReactDOM.render(<App />, MOUNT_NODE);
-};
+const App = () => (
```

Comment:
I like the new page!

Reply:
I thought that replicating `chart.js` would make it more clear for users what this library is for/capable of.